src/StudentList.java

```diff
@@ -21,4 +21,37 @@ public static Student getStudentById(int id) {
                 .orElse(null);
     }
 
+    public static Student getStudentByIdV1(int id) {
+
+        Student student1 = students.stream()
+                .filter(student -> student.getId() == id)
+                .findFirst()
+                .orElse(null);
+        if (student1 == null) throw new RuntimeException("Student Not Found");
+
+        return student1;
+    }
+
+    public static Student getStudentByIdV2(int id) throws Exception {
```

Comment:
It is Not good to use Exception..make it more specific..you can have your own checked exception